### PR TITLE
Fixes to viz.yaml

### PR DESCRIPTION
**Pull request by jiwalker-usgs**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,5 +3,7 @@ target/
 user/
 figures/
 vizlab/
-makefile
+[Mm]akefile
 .Rproj.user
+.Rhistory
+internal-analytics.Rproj
```
Comment (ldecicco-USGS) [inline .gitignore]: so as we're learning, if we don't ignore internal-analytics.Rproj, we might have a better time unifying everyone's tab/space issue....

Comment (ldecicco-USGS) [inline .gitignore]: Right. I think I'm saying the same thing. Our group has traditionally ignored it, but this:

image

Might be a good reason for us to reverse that policy

**Comment by jiwalker-usgs** [comment]
I've committed my .Rproj file, but I think it might include things other people don't normally set.

**Comment by ldecicco-USGS** [comment]
I've got knitr instead of Sweave, that might be the one thing I'd actually recommend changing.

Otherwise, you have:
```
AutoAppendNewline: Yes
 +StripTrailingWhitespace: Yes
```
which I see no reason *not* to include....just not something that's in the default.

**Comment by jiwalker-usgs** [comment]
Ok, it shouldn't matter for vizlab projects, but I've switched to knitr.

**Pull request merged**